src/wireless-security/ws-wpa-eap.c

```diff
@@ -99,7 +99,8 @@ update_secrets (WirelessSecurity *parent, NMConnection *connection)
 WirelessSecurityWPAEAP *
 ws_wpa_eap_new (NMConnection *connection,
                 gboolean is_editor,
-                gboolean secrets_only)
+                gboolean secrets_only,
+                const char **secrets_hints)
```

Comment:
wireless-security is an internal library, both used by libnma and libnm-gtk. You didn't adjust libnm-gtk, so this breaks build `--with-libnm-gtk`.

Maybe use type `const char *const*`?

Reply:
Yep, will fix both things.

Reply:
Regarding modifying libnm-gtk it seems it is in use by external projects (only gnome-control-center that I could find), would it be better to leave the nma_wifi_dialog_new signature as is and add a new call nma_wifi_dialog_new_for_secrets?

And if this is done in libnm-gtk should the same be done in libnma?

Reply:
libnm-gtk is only on life-support, you don't need add new functionality there at all. Whatever is the least effort. Yes, a new `nma_wifi_dialog_new_for_secrets()` sounds good.